cluster/images/etcd/Makefile

```diff
@@ -34,19 +34,25 @@ LATEST_ETCD_VERSION?=3.2.24
 # REVISION provides a version number fo this image and all it's bundled
 # artifacts. It should start at zero for each LATEST_ETCD_VERSION and increment
 # for each revision of this image at that etcd version.
-REVISION?=0
+REVISION?=1
 
 # IMAGE_TAG Uniquely identifies k8s.gcr.io/etcd docker image with a tag of the form "<etcd-version>-<revision>".
 IMAGE_TAG=$(LATEST_ETCD_VERSION)-$(REVISION)
 
 ARCH?=amd64
+ALL_ARCH = amd64 arm arm64 ppc64le s390x
 # Image should be pulled from k8s.gcr.io, which will auto-detect
 # region (us, eu, asia, ...) and pull from the closest.
 REGISTRY?=k8s.gcr.io
 # Images should be pushed to staging-k8s.gcr.io.
 PUSH_REGISTRY?=staging-k8s.gcr.io
+
+MANIFEST_IMAGE := $(PUSH_REGISTRY)/etcd
+
+# This option is for running docker manifest command
+export DOCKER_CLI_EXPERIMENTAL := enabled
 # golang version should match the golang version from https://github.com/coreos/etcd/releases for the current ETCD_VERSION.
-GOLANG_VERSION?=1.8.7
+GOLANG_VERSION?=1.11
```

Comment:
/hold

This looks wrong? https://github.com/etcd-io/etcd/blob/34fcabab55585b39835820278da4de65b3463358/CHANGELOG-3.2.md#v3224-2018-07-24

Reply:
good catch, let's switch this back to 1.8.7 since that's the recommendation

Reply:
/hold cancel

I pushed another commit reverting this